digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -566,6 +565,7 @@ RunTaskRequest buildRunTaskRequest(
         setEcsTaskOverride(commandContext, commandRequest, td, runTaskRequest, clientConfig); // RuntimeException,ConfigException
         setEcsTaskLaunchType(clientConfig, runTaskRequest);
         setEcsTaskStartedBy(clientConfig, runTaskRequest);
+        setEcsTaskTags(clientConfig, runTaskRequest);
```

Comment:
We'd better write a test to see if `RunTaskRequest.tags` is expectedly set?

Reply:
@komamitsu 
Actually it is difficult to test `setEcsXXX` methods themselves because they are protected methods and `runTaskRequest` is a just local variable. So it is difficult to evaluate them.

It is the reason why I tested the setup configuration logic at `EcsClientConfigTest`. https://github.com/treasure-data/digdag/pull/1517/files#diff-c0efef3822af1f3097d11b328b9c7f1044e8bf176b73791a02e819c19979fcdeR126-R161

If you have a good idea about this testing, could you tell me it?

Reply:
I think adding a test in `io.digdag.standards.command.EcsCommandExecutorTest` using some mocks is an option. We may be able to add some tests for this change later.